platform-observability-agent/debian/postinst

```diff
@@ -46,6 +46,16 @@ chown platform-observability-agent:bm-agents /opt/telegraf/bin/collect_gpu_metri
 chown platform-observability-agent:bm-agents /opt/telegraf/bin/collect_disk_info.sh
 chown platform-observability-agent:bm-agents /opt/telegraf/bin/core_metrics.sh
 
+chmod 744 /etc/edge-node/node/confs/platform-observability-collector
```

Comment:
We could have limited to 740 or even 700 as I see specific user access configured

Reply:
I've updated to 740 for access